CONTRIBUTING.md

```diff
@@ -255,6 +255,8 @@ def load_audio(audio_path):
     # -- for example, the code below. This should be dataset specific!
     # -- By default we load to mono
     # -- change this if it doesn't make sense for your dataset.
+    if not os.path.exists(audio_path):
+        raise IOError("audio_path {} does not exist".format(audio_path))
```

Comment:
what if the file exists, but we don't have read permission?

Reply:
also, what if it exists, has read permissions, but has the wrong checksum?

Reply:
> what if the file exists, but we don't have read permission?

We won't catch it here, but the function will still error later when we open the file, just with a less friendly error message. I think it's an unusual enough case that I'm OK with not being 100% exhaustive of all possible cases. 

> also, what if it exists, has read permissions, but has the wrong checksum?

We used to validate on load, but it was _very_ slow and made the library much less usable. @magdalenafuentes and I made the sad (but in our opinion) necessary decision to leave validation up to the user. This is what `dataset.validate()` is for!